### ISSUE TITLE
Change image_size to 256

### ISSUE THREAD

**Issue by viettmab** [rosinality/glow-pytorch#30]
Hi,
I want to change image_size to 256. Do I need to change the value of n_bits and n_bins?
Another question I want to ask is the meaning of the code from lines 111 to 116 in https://github.com/rosinality/glow-pytorch/blob/master/train.py#L111
Thank you
Viet Nguyen

**Comment by rosinality** [comment]
In the paper authors used 5 bits for 256px.
And `image * 255` is to rescale value ranges for `args.n_bits < 8`, as `image` is in [0, 1].

**Comment by viettmab** [comment]
In the https://github.com/rosinality/glow-pytorch/blob/master/train.py,
After line 109 image is in [0,1]. After line 111 image is in [0,255]. After line 114 image is in [0,31.875]. After line 116, image is in [-0.5,0.5]. Am I right? Why don't we just change from [0,1] to [-0.5,0.5] directly?

**Comment by rosinality** [comment]
No, it is different as it is multiplied with 255 (not 256) and floor is applied.

**Comment by viettmab** [comment]
Okay, I understand. So I change the image_size to 256 instead of 64 so that the image size becomes (3,256,256). Do I need to change any value, e.g n_bits,..?

**Comment by rosinality** [comment]
I think it is safe to use the settings in the paper. (5 bits)

**Comment by viettmab** [comment]
Yeah, thank you very much!

**Comment by Moleculebo** [comment]
Hi, when I change  the image_size to 256 instead of 64 so that the image size becomes (3,256,256), some errors occured, can you share your code?
